src/Controller/Menu/StoreController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Controller\Menu;
+
+use App\Entity\Menu;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class StoreController
+{
+    public function __invoke(Request $request, Container $container)
+    {
+        $entityManager = $container->get('entity.manager');
+        $name = $request->get('name', '');
```

Comment:
wouldn't have been better to throw a RuntimeException or to set default value or random value instead of nothing?